chef/cookbooks/neutron/recipes/common_config.rb

```diff
@@ -120,7 +120,10 @@
   service_plugins = "#{service_plugins}, neutron.services.loadbalancer.plugin.LoadBalancerPlugin"
 end
 
-network_nodes = search_env_filtered(:node, "roles:neutron-network")
+network_nodes_count = 1
```

Comment:
I'd prefer to initialize this with `neutron[:neutron][:elements]["neutron-network"].count` (which should be 1 anyway, but is more future-proof in case there are some changes later on).